agent_central/services/hq_service.py

```
@@ -298,41 +352,156 @@ def consolidate_knowledge(self):
+                quality = {"updated_at": None, "skills": {}}
+
+        skills = quality.get("skills", {})
+        events = feedback_log.read_text(encoding="utf-8").splitlines()
+        if not events:
+            return
+
+        for line in events:
+            try:
+                event = json.loads(line)
+            except Exception:
+                continue
```

Ruff (0.14.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)